client/src/components/Crawler/igdb.js

```diff
@@ -0,0 +1,270 @@
+import axios from 'axios';
+const USER_KEY = 'fbf8c51fc8298662fc1057f0e26c2399'
+const HEADER = {
+    'Accept': 'application/json',
+    'user-key': USER_KEY
+}
+const DEFAULT_COVER_PATH = "/img/gg-logo.png";
+const PROXY = "https://evening-scrubland-53471.herokuapp.com/";
+const DOMAIN = "https://api-v3.igdb.com/";
+
+// Helper function to make HTTP Requests to IGDB
+async function igdbAPI({ endpoint, data }) {
+    // console.log("Calling " + endpoint + " with " + data)
+    return axios({
+        url: PROXY + DOMAIN + endpoint,
+        method: 'POST',
+        headers: HEADER,
+        data: data
+    });
+}
+
+// cover query
+function coverQuery({ gameID }) {
+    return igdbAPI({
+        endpoint: 'covers',
+        data: 'fields image_id; where game = ' + gameID + ';'
+    });
+}
+
+//gets the first 10 games with matching text
+export async function getPreviews({ searchText }) {
+    let gameData = {};
+    try {
+        const response = await igdbAPI({
+            endpoint: 'games',
+            data: 'search "' + searchText + '"; fields id, name, summary, url; limit 5;'
+        });
+        for (var i in response.data) {
+            gameData[i] = {};
+            gameData[i].id = response.data[i].id;
+            gameData[i].name = response.data[i].name;
+            gameData[i].summary = response.data[i].summary;
+            gameData[i].url = response.data[i].url;
+        }
+        const numGames = response.data.length;
+        let queryArray = [];
+        for (let i_1 = 0; i_1 < numGames; i_1++) {
+            queryArray.push(coverQuery({ gameID: gameData[i_1].id }));
+        }
+        const responseArray = await axios.all(queryArray);
+        for (let i_2 = 0; i_2 < responseArray.length; i_2++) {
+            if (responseArray[i_2].data[0]) {
+                gameData[i_2].coverURL = "https://images.igdb.com/igdb/image/upload/t_cover_big/"
+                    + responseArray[i_2].data[0].image_id + ".jpg";
+            }
+            else {
+                gameData[i_2].coverURL = DEFAULT_COVER_PATH;
+            }
+        }
+        gameData.success = true;
+        return gameData;
+    }
+    catch (err) {
+        console.error(err);
+        return {
+            success: false
+        };
+    }
+}
+
+// retrieves background image url given game id
+async function getBackground({ gameID }) {
+    try {
+        const response = await igdbAPI({
+            endpoint: 'screenshots',
+            data: 'fields image_id; where game = ' + gameID + ';'
```

Comment:
```suggestion
            data: `fields image_id; where game = ${gameID};`
```
A more concise way of embedding variables into strings.